common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1017,6 +1052,98 @@
                 }
             },
 
+            handleCaptioningCookie: function() {
```

Comment:
Is the `.cookie` method part of jquery by default or does it need to be "required"?